_posts/2017-06-02-jslang-meetup-async-await.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Async Function Kata at #jslang Meetup"
+date:   2017-06-02 08:03:00 +0200
+categories: meetup javascript kata
+---
+
+The callback-hell times are long over. We have better tools now. But which of the new kids on the block do you want to 
+play with? In this meetup, last night we tried out [async functions][async-funcs], also known by the name "async/await". 
```

Comment:
The fact that you can somehow **deduce** when the meetup actually happened is really besides the point - to me personally it fells stylistically "odd" but that is only a suggestion and others including @wolframkriesing might disagree - which is fine ;-)